engine/src/main/java/org/terasology/config/flexible/FlexibleConfigImpl.java

```diff
@@ -138,7 +136,7 @@ public void load(Reader reader) {
 
             jsonReader.endObject();
         } catch (Exception e) {
-            // TODO: Handle exception
+            throw new RuntimeException("Error parsing config file!");
```

Comment:
Hmmm... I imagine this prints out the whole stack trace normally. But will this include any hint of where the parsing went wrong? It would be important to have that information printed out.

And can the catch any more precise? I.e. see:

https://www.javaworld.com/article/2077500/testing-debugging/java-tip-134--when-catching-exceptions--don-t-cast-your-net-too-wide.html

Reply:
I restructured the code slightly to print the name of the config that failed to parse, and printed the stack trace of that exception as well - that should cover the "where" part.